tests/integration/clientv3/lease/leasing_test.go

```diff
@@ -23,8 +23,9 @@ import (
 	"testing"
 	"time"
 
+	"github.com/stretchr/testify/assert"
```

Comment:
Nitpick: let's sort this after all of etcd own imports.

Reply:
VSCode is preferring this import order though with testify/assert sitting on top. Every time I save the file with the new import at the bottom, it shifts back up :/

Reply:
No worries was a nit!